conf/invenio-local.conf-example

```diff
@@ -223,3 +223,5 @@ CFG_OAI_METADATA_FORMATS = {
     'oai_dc': ('XOAIDC', 'http://www.openarchives.org/OAI/1.1/dc.xsd', 'http://purl.org/dc/elements/1.1/'),
     'emarcxml': ('XOAIMARCE', 'http://www.loc.gov/standards/marcxml/schema/MARC21slim.xsd', 'http://www.loc.gov/MARC21/slim'),
     }
+
+CFG_GOOGLE_MAPS_API_KEY = 'AIzaSyBdVl-cTICSwYKrZ95SuvNw7dbMuDt1KG0'
```

Comment:
Not a real key, comes from https://developers.google.com/places/android-api/signup.